src/bentoml/_internal/frameworks/transformers.py

```diff
@@ -750,10 +751,29 @@ def import_model(
         resume_download=resume_download,
         **extra_hf_hub_kwargs,
```

Comment:
shouldn't trust_remote_code will also have to pass it here as well?

Reply:
trust_remote_code is packaged into extra_hf_hub_kwargs here. Alternatively, if you think trust_remote_code should be explicitly declared as a keyword arg, I can make that change as well.

Reply:
it should